drivers/sensor/si7006/Kconfig

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2019 Vaishali Pathak <vaishali@electronut.in>
```

Comment:
Usually your employer is assigned copyright when you contribute with a company email. Should this be:
```
Copyright (c) 2019 Electronut Labs
```
?